### PR TITLE
Points calculation

### PR DESCRIPTION
**Pull request by scolear**

### CODE REVIEW

File: config.devnet.json [hunk 1]
```diff
@@ -22,5 +22,13 @@
       "address": "0x0DD4f29E21F10cb2E485cf9bDAb9F2dD1f240Bfa",
       "logo": "/images/logos/dlc-link-logo-horizontal.svg"
     }
+  ],
+  "protocols": [
+    {
+      "name": "Curve",
+      "lpTokenAddress": "0xe957ce03ccdd88f02ed8b05c9a3a28abef38514a",
+      "gaugeAddress": "0x02b8e750e68cb648db2c2ac4bbb47a10a5c12588",
+      "logo": "/images/logos/dlc-link-logo-horizontal.svg"
```
Comment (scolear) [inline config.devnet.json:31]: remove or update

File: netlify/functions/fetch-user-address-risk.ts [hunk 2]
```diff
@@ -28,4 +48,6 @@ export async function handler(event) {
       message: await response.json(),
     }),
   };
-}
+};
```
Comment (Polybius93) [inline netlify/functions/fetch-user-address-risk.ts:51]: Nice update!

File: src/app/components/proof-of-reserve/proof-of-reserve.tsx [hunk 3]
```diff
@@ -33,18 +33,20 @@ export function ProofOfReserve(): React.JSX.Element {
...
-  async function fetchAllMintBurnEventsHandler(): Promise<ProtocolHistoryTableItemProps[]> {
-    const detailedEvents = await fetchAllMintBurnEvents();
+  async function fetchMintBurnEventsHandler(): Promise<ProtocolHistoryTableItemProps[]> {
+    const detailedEvents = await fetchMintBurnEvents(undefined, 10);
     return detailedEvents.map((event, index) => {
       const isMint = event.from.toLowerCase() === BURN_ADDRESS.toLowerCase();
+      // const knownMerchant = appConfiguration.merchants.find(
+      //   merchant => merchant.address === (isMint ? event.to : event.from)
+      // );
+      // console.log('knownMerchant', knownMerchant);
```
Comment (Polybius93) [inline src/app/components/proof-of-reserve/proof-of-reserve.tsx:45]: Remove these comments if they are not necessary to have.

**Review by Polybius93 — APPROVED** [review]
Nice job, looks good!

File: netlify/functions/fetch-user-address-risk.ts (hunk 2 above)
Comment (Polybius93) [reply netlify/functions/fetch-user-address-risk.ts:51]: I mean the typescript change

**Pull request merged**